styles.css

```diff
@@ -0,0 +1,265 @@
+    -webkit-border-radius: 0.3rem;
+    -moz-border-radius: 0.3rem;
+    border-radius: 0.3rem;
+}
+.item-row {
+    color: #333;
+}
+.current {
+    background: rgba(243,167,116, 0.2);
+    color: #F3A774;
+    border-bottom: 4px solid #F3A774;
+}
+.tabs {
+    display: none;
+    border-radius: 8px;
+    overflow: hidden;
+}
+#tab {
+    position: absolute;
+    opacity: 0;
+    z-index: -1;
+}
+.tab {
+    width: 100%;
+    overflow: hidden;
+}
+.tab-label {
+    display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+    display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+    display: -ms-flexbox;      /* TWEENER - IE 10 */
+    display: -webkit-flex;
+    display: flex;
+    justify-content: space-between;
+    -webkit-justify-content: space-between;     /*iOS*/
+    -ms-flex-pack: justify;
+    padding: 1em;
+    font-weight: bold;
+    cursor: pointer;
+}
+.tab-label:hover {
+    cursor: pointer;
+}
+.tab-label::after {
+    content: "\276F";
+    width: 1em;
+    height: 1em;
+    text-align: center;
+    transition: all .35s;
+}
+.tab-content {
+    max-height: 0;
+    padding: 0 1em;
+    color: #2c3e50;
+    background: white;
+    transition: all .35s;
+}
+/*.tabs input:checked + .tab-label {*/
+/*    background: #1a252f;*/
+/*}*/
+.tabs input:checked + .tab-label::after {
+    -webkit-transform: rotate(90deg);
+    transform: rotate(90deg);
+}
+.tabs input:checked ~ .tab-content {
+    max-height: 100vh;
+    padding: 1em;
+}
+form.big-form, .tabs form {
+    display: -webkit-box;      /* OLD - iOS 6-, Safari 3.1-6 */
+    display: -moz-box;         /* OLD - Firefox 19- (buggy but mostly works) */
+    display: -ms-flexbox;      /* TWEENER - IE 10 */
+    display: -webkit-flex;
+    display: flex;
+    height: 100px;
+    align-items: center;
+    -ms-flex-align: center;
+    justify-content: space-evenly;
+    -ms-flex-pack: justify;
+    -webkit-justify-content: space-between;    /* iOS*/
+    padding: 0 1rem;
+}
+form.big-form p, form.big-form input, form.big-form select, form.big-form input[type=submit],
+.tabs form p, .tabs form input, .tabs form select, .tabs form input[type=submit] {
+    color: #1A294E;
+}
+form.big-form input, form.big-form select,
+.tabs form input, .tabs form select {
+    border: 1px solid #ededed;
+}
+form.big-form p,
+.tabs form p {
+    font-weight: 700;
+}
+form.big-form input,
+.tabs form input {
+    padding: 0.5rem;
+}
+form.big-form select,
+.tabs form select,
+form.big-form input,
+.tabs form input {
+    width: 189px;
+    height: 38px;
+    background: #fff;
+}
+form.big-form input[type=submit],
+.tabs form input[type=submit] {
+    background: #F3A774;
+    width: 100px;
+    border: none;
+    -webkit-border-radius: 0.3rem;
+    -moz-border-radius: 0.3rem;
+    border-radius: 0.3rem;
+}
+form.big-form input[type=submit]:hover,
+.tabs form input[type=submit]:hover {
+    cursor: pointer;
+}
+.PANTRY-row, .FRIDGE-row, .FREEZER-row, .PRODUCE-row, .OTHER-row {
```

Comment:
Classes should be all lower case

Reply:
this class name is assigned by $_GET, so I cannot change it to lower case